scripts/ci/test-check-target-tree.py

```
@@ -203,20 +203,33 @@ def test_exception_row_for_a_crate_now_at_its_section5_path_fails(self) -> None:
-            code, output = self.run_gate()
+        victim["manifest_path"] = victim["manifest_path"].replace(
+            "crates/loop/ironclaw_agent_loop", "crates/kernel/ironclaw_agent_loop"
+        )
+        stale = GATE.Exception_(
+            package="ironclaw_agent_loop",
+            actual="crates/somewhere_else",
+            documented="crates/loop/ironclaw_agent_loop",
+            owner="self-test synthetic row",
+            why="synthesized by the self-test; never a real disposition",
+        )
+        with mock.patch.object(GATE, "EXCEPTIONS", GATE.EXCEPTIONS + (stale,)):
```

Ruff (0.16.1)

[warning] Consider `(*GATE.EXCEPTIONS, stale)` instead of concatenation

Replace with `(*GATE.EXCEPTIONS, stale)`

(RUF005)